src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -913,6 +914,84 @@
     ]
 }
 
+ANALYTICS_CREDENTIAL_OAUTH_INPUTS = {
+    "fields": [
+        {
+            "id": "auth_type",
+            "label": "Analytics Authentication Type",
+            "type": "string",
+            "default": "oauth",
+            "hidden": True,
+        },
+        {
+            "id": "client_id",
+            "label": "Client ID",
+            "type": "string",
+            "help_text": ("The Client ID from the Authorization Server."),
+        },
+        {
+            "id": "client_secret",
+            "label": "Client Secret",
+            "type": "string",
+            "secret": True,
+            "help_text": ("The Client Secret from the Authorization Server."),
+        },
+        {
+            "id": "gather_interval",
+            "label": "Analytics Gather Interval",
+            "type": "string",
+            "help_text": "The time interval between each collection (secs)",
+            "default": "14400",
+        },
+        {
+            "id": "insights_tracking_state",
+            "label": "Enable Red Hat Cloud Analytics",
+            "type": "boolean",
+            "default": False,
+        },
+    ],
+    "required": ["auth_type", "client_id", "client_secret"],
+}
+
+ANALYTICS_CREDENTIAL_BASIC_INPUTS = {
+    "fields": [
+        {
+            "id": "auth_type",
+            "label": "Analytics Authentication Type",
+            "type": "string",
+            "default": "basic",
+            "hidden": True,
+        },
+        {
+            "id": "username",
+            "label": "Username",
+            "type": "string",
+            "help_text": "The username of REDHAT or SUBSCRIPTIONS",
+        },
+        {
+            "id": "password",
+            "label": "Password",
+            "type": "string",
+            "secret": True,
+            "help_text": "The password of REDHAT or SUBSCRIPTIONS",
+        },
+        {
+            "id": "gather_interval",
+            "label": "Analytics Gather Interval",
+            "type": "string",
+            "help_text": "The time interval between each collection (secs)",
+            "default": "14400",
+        },
+        {
+            "id": "insights_tracking_state",
+            "label": "Enable Red Hat Cloud Analytics",
```

Comment:
Both controller and gateway have setting page for this parameter, although they used different labels. We should not introduce the third label which brings more confusion to users. We should either align with gateway or controller.
Gateway label: Insights tracking state. Help_text: Enables the service to gather data on automation and send it to Automation Analytics.
Controller label: Gather data for Automation Analytics. No help text.
I prefer to align with gateway because eventually we will use the settings from gateway.

Reply:
@bzwei Both places are updated accordingly